lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -891,13 +968,13 @@ def _orchestrators(
         cm_relations = [
             rel.id
             for rel in self.model.relations[self.relation_name]
-            if rel.id != event.relation.id
+            if rel.id != event.relation.id and rel.units
```

Comment:
let's be more explicit here when mixing boolean and implicit size checks, perhaps:
```suggestion
            if rel.id != event.relation.id and len(rel.units) > 0
```